source/mainnet/smart-contracts/tutorials/index.rst

```diff
@@ -15,6 +16,10 @@ You are going to write a basic web front-end example that can read from and writ
 
 In the ``Minting`` tutorials you will mint various types of tokens.
 
+In the ``eSealing`` tutorial, you are going to learn how to seal a document, so you can prove that it was in your possession at the time of sealing.
```

Comment:
I think we should be more considerate about where we use the different types of text highlighting. In my mind, we should only use the `` when referring to code or terminal content.
Besides, adding direct links to the tutorials in each summary section here makes it easier for the user to get to the one they are interested in (right now they have to read the text, then find the correct link in the list below).